config/development.json

```diff
@@ -81,7 +81,6 @@
 		"jetpack/connect-redirect-pressable-credential-approval": true,
 		"jetpack/connect/remote-install": true,
 		"jetpack/connect/mobile-app-flow": true,
-		"jetpack/connect/woo-dna": true,
```

Comment:
Does this change make WooDna flow available in prod?

Reply:
Yes. Before, Calypsohad the "map" of services that used Woo DNA, so it could be short-circuited with a feature flag ([here](https://github.com/Automattic/wp-calypso/pull/43282/files#diff-dcd22633c51427adff3176462faf6051L6)). Now, wp.com server-side logic is controlling that instead, so there's no point of having a client-side feature flag. If there's no `woodna_*` arguments in the querystring, then the normal flow will be used instead.

Reply:
I like that approach more. Thank you!